uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/request-modal-details-toil.controller.js

```diff
@@ -11,7 +11,12 @@ define([
   function RequestModalDetailsToilController ($log, $q, $rootScope, OptionGroup, AbsenceType, detailsController) {
     $log.debug('RequestModalDetailsToilController');
 
-    detailsController.requestCanExpire = true;
+    var canExpireAccordingTo = {
```

Comment:
Names like `previousExpirationDateValue` or `adminSettings` does not denote they are boolean in nature which might be confusing. I suggest following names or something similar.

```
expirationFieldVisibleAsIt = { hasPreviousExpirationDate: ... , hasAdminSettings: ... }
```